app/models/ability.rb

```diff
@@ -19,119 +19,60 @@ def initialize(user)
     # here are :read, :create, :update and :destroy.
     #
     # The second argument is the resource the user can perform the action on.
-    # If you pass :all it will apply to every resource. Otherwise pass a Ruby
+    # If you pass :all, it will apply to every resource. Otherwise, pass a Ruby
     # class of the resource.
     #
     # The third argument is an optional hash of conditions to further filter the
-    # objects.
-    # For example, here the user can only update published articles.
-    #
+    # objects. For example, here the user can only update published articles.
     #   can :update, Article, :published => true
     #
+    # The block argument takes as a parameter an instance of the object for which
+    # permission's being checked. If the block returns true, the user is granted that
```

Comment:
i think it would be better to say `permission is` here just to be even clearer (as the added commas above make things clearer)